ocs_ci/utility/utils.py

```diff
@@ -1129,27 +1129,67 @@ def get_testrun_name():
     markers = config.RUN['cli_params'].get('-m', '').replace(" ", "-")
     us_ds = config.REPORTING.get("us_ds")
     us_ds = "Upstream" if us_ds.upper() == "US" else "Downstream" if us_ds.upper() == "DS" else us_ds
+    ocp_version = ".".join(
+        config.DEPLOYMENT.get('installer_version').split('.')[:-2]
+    )
+    ocp_version_string = f"OCP{ocp_version}" if ocp_version else ''
+    # todo: use provided --ocsci-version instead of testrun_name_part
+    #  once it is available
+    testrun_name_part = config.REPORTING.get('testrun_name_part')
+    try:
+        testrun_name_part = f"OCS{float(testrun_name_part)}"
```

Comment:
@petr-balogh this isn't forcing a float value, it's attempting to convert it to a float. If it can(in the event we are passing a version through here), we prepend `OCS` to the string and use that. Otherwise we leave it as is.

Reply:
OK, I thought that if this will fail:  f"OCS{float(testrun_name_part)}" and raise an exception that you will not have that value but actually you will from line 1138.

I would like to completely get rid of those parameters: 
`testrun_name_part` and `testrun_name` and the best case use just one clear parameter for this purpose:
`display_name` 

or maybe we don't need display_name and we can use just `testrun_name` instead which will make things clear with just one parameter?